engine.jackson/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,9 @@
+log4j.rootLogger=DEBUG, STDOUT
```

Comment:
We've generally tried to keep only slf4j dependencies in the `main` modules. We've used `slf4j-simple` for tests. Most of the downstream projects are actually using `logback-classic`

Reply:
Deleted from the module